.github/workflows/build.yml

```diff
@@ -388,35 +339,23 @@ jobs:
           shell: bash
       steps:
         - uses: actions/checkout@v4
-        - name: Setup ccache
-          uses: hendrikmuhs/ccache-action@v1.2
-          with:
-            key: mingw-ccache
```

Comment:
Removed ccache usage since the key was getting duplicated and therefor getting poor utilization. The loader doesn't really need it anyway.